module-apps/application-meditation/widgets/TimerProperty.cpp

```diff
@@ -118,6 +122,32 @@ std::chrono::minutes TimerProperty::getTime() noexcept
     return state.getTime();
 }
 
+bool TimerProperty::setTime(std::int32_t newValue)
+{
+    bool result = state.setTime(newValue);
+    setMeditationTime();
+    return result;
+}
+void TimerProperty::setOnChangeCallback(OnChangeCallback callback)
+{
+    onChangeCallback = callback;
+}
+
+bool TimerProperty::State::setTime(int value)
+{
+    bool result   = true;
+    const auto it = std::find(
+        std::begin(Constants::Params::meditationDurations), std::end(Constants::Params::meditationDurations), value);
+    if (it != std::end(Constants::Params::meditationDurations)) {
+        timeInMinutes = *it;
```

Comment:
You could just `return true` here and avoid `bool result` and the `else` part

Reply:
done